dashboard/app/models/pd/workshop_daily_survey.rb

```diff
@@ -62,26 +64,45 @@ def set_day_from_form_id
       :day
     )
 
-    VALID_DAYS = (0..5).freeze
+    # Different categories have different valid days
+    VALID_DAYS = {
+      LOCAL_CATEGORY => (0..5).to_a.freeze,
+      ACADEMIC_YEAR_1_CATEGORY => [1].freeze,
+      ACADEMIC_YEAR_2_CATEGORY => [1].freeze,
+      ACADEMIC_YEAR_3_CATEGORY => [1].freeze,
```

Comment:
If Q3 workshops are more than 1 day long, this needs to change. Same for Q4 and Q3Q4 workshops